cmake/GNDStk_dependencies.cmake

```diff
@@ -28,12 +28,20 @@ FetchContent_Declare( pugixml-adapter
     GIT_SHALLOW     TRUE
     )
 
-FetchContent_Declare( json-adapter
-    GIT_REPOSITORY  http://github.com/njoy/json-adapter
-    GIT_TAG         origin/master
-    GIT_SHALLOW     TRUE
+FetchContent_Declare(json
+    GIT_REPOSITORY https://github.com/nlohmann/json.git
+    GIT_TAG v3.7.3
+    GIT_SHALLOW true
     )
 
+FetchContent_GetProperties(json)
+set(JSON_BuildTests OFF CACHE INTERNAL "")
+if(NOT json_POPULATED)
+  FetchContent_Populate(json)
+  add_subdirectory(${json_SOURCE_DIR} ${json_BINARY_DIR} EXCLUDE_FROM_ALL)
+endif()
```

Comment:
Is this just to turn off the tests?  Or is there something else that broke from the canonical pattern?

Reply:
I'm not sure. This was just the instructions that was given on the [nlohmann::JSON website](https://github.com/nlohmann/json#embedded-fetchcontent).